taskcluster/config.yml

```diff
@@ -52,6 +52,11 @@ workers:
             implementation: generic-worker
             os: windows
             worker-type: b-win2022
+        b-win11-2024h2-arm64:
```

Comment:
RN we only have level-1 Workers, we should followup with releng, that we also get the level-3 queue up before we merge this, otherwise those tasks will be stuck :)